cactusbot/commands/magic/social.py

```diff
@@ -0,0 +1,46 @@
+"""Get social data."""
+
+import aiohttp
+
+from . import Command
+
+from ...packets import MessagePacket
+
+
+class Social(Command):
```

Comment:
This is Beam-specific, not service-agnostic. However, that will be fixed with #60.